### ISSUE TITLE
vendoring a project structure with gb doesn't work

### ISSUE THREAD

**Issue by stunti** [constabulary/gb#352]
I have a project which is structured with gb. It works perfectly on its own. I need to include a package of that project in another project.
When I do that, because in the first project, path imported are in the form of "cmd/back/global" and not "github.com/stunti/back/global" gb doesn't find the package once included in the second project.

FATAL command "build" failed: failed to resolve import path "cmd/cli/tasks": cannot find package "cmd/back/global" in any of:

Is there a special way to do it or I have to copy everything?

**Comment by davecheney** [comment]
Can you please tell me more about the project you are including this project in ? Is this a gb project or are you trying to go get it ?

**Comment by stunti** [comment]
This is a project which is already structured to work with gb. 
I did run gb vendor to get it and I am trying to import one of its package.

Inside the package from the first project it use other packages from the first project too which are in the form of "cmd/pk1/pk2". The issue is that gb when trying to build the second project doesn't find these packages because they do not refer to any known path in the vendor list of the second package.

Not sure if it is really clear

**Comment by stunti** [comment]
That's the basic structure

Remote1
```
├── src
│   └── cmd
│       └── back
│           ├── lib
│           │   └── lib.go (include util package using "cmd/back/util")
│           │
│           └── util
│               └── util.go
```


Big1
```
├── src
│    └── cmd
│        └── helloworld
│            └── helloworld.go (include github.com/stunti/remote1/src/cmd/back/lib)
│
│               
└── vendor
    └── src
        └── github.com
            └── stunti
                 └── remote1	
                    └── src
                        └── cmd
                            └── back
                                ├── lib
                                │   └── lib.go
                                │
                                └── util
                                    └── util.go (include util package using "cmd/back/util")
```

**Comment by davecheney** [comment]
At the moment gb vendor assumes the code your are vendoring is in go get format. It shouldn't, but it does.  The infrastructure is there to support gb vendoring gb projects, but it is not done yet.

**Comment by davecheney** [comment]
Merged into #443
